include/Reader/reader.h

```diff
@@ -3241,6 +3251,13 @@ class ReaderBase {
   virtual bool arrayGet(CORINFO_SIG_INFO *Sig, IRNode **RetVal) = 0;
   virtual bool arraySet(CORINFO_SIG_INFO *Sig) = 0;
 
+  /// \brief Check whether structs are represented by pointers on the operand
+  /// stack.
+  ///
+  /// \returns true iff structs are represented by pointers on the operand
+  /// stack.
+  virtual bool structsAreRepresentedByPointers() = 0;
```

Comment:
I agree with Andy. `ReaderBase` shouldn't need to know how values are represented by its subclasses. This does feel like something we can fix up later, however.